tests/test-common.js

```diff
@@ -391,6 +424,40 @@ describe('ocapld.js', () => {
         expect(result.verified).to.be.true;
       });
 
+      it('should NOT verify a root capability w/ separate target when ' +
```

Comment:
I think this test suite is saying 'should fail' as with the other test you added.

Reply:
Addressed here: https://github.com/digitalbazaar/ocapld.js/pull/25/commits/9891ee8431b0035ca8b7f734831abd31e71b82a4

This PR is not priority or essential so feel free to nit pick these if you want:

```
  ocapld.js
    Common
      sign with capabilityInvocation proof purpose
        ✓ should succeed w/key invoker (44ms)
        ✓ should succeed w/controller invoker
        ✓ should fail when missing "capability"
      sign with capabilityDelegation proof purpose
        ✓ should succeed w/key delegator
        ✓ should succeed w/controller delegator
        ✓ should fail when missing "capabilityChain" and "parentCapability"
    Verifying capability chains
      Invoker and Delegator as keys
        ✓ should verify a self-invoked root capability
        ✓ should verify w/array expected target
        ✓ should verify a capability chain of depth 2
        ✓ should fail to verify a capability chain of depth 2when the expectedRootCapability does not match
        ✓ should verify invoking a capability chain of depth 2
      Invoker and Delegator as controllers
        ✓ should verify a self-invoked root capability
        ✓ should fail to verify a root capability as delegated
        ✓ should verify a self-invoked root capability with missing invoker and delegator
        ✓ should verify a invoking root capability w/ separate target when a matching `expectedRootCapability` is given
        ✓ should verify a root capability w/ separate target when a matching `expectedRootCapability` array is given
        ✓ should fail to verify a root capability w/ separate target when `expectedRootCapability` is not a URI
        ✓ should fail to verify a root capability w/ separate target when no `expectedRootCapability` is given
        ✓ should verify a root capability w/ multiple controllers
        ✓ should verify a root capability w/ multiple invokers
        ✓ should verify a capability chain of depth 2
        ✓ should verify invoking a capability chain of depth 2
        ✓ should fail if expectedRootCapability does not match
        ✓ should fail if target does not match root ID
        ✓ should verify a capability chain of depth 2 and a valid caveat on one capability
        ✓ should fail to verify a capability chain of depth 2 when a caveat is not met on one capability
        ✓ should verify a capability chain of depth 2 and an allowed action on one capability
        ✓ should fail to verify a capability chain of depth 2 when a "capabilityAction" is required but missing
        ✓ should fail to verify a capability chain of depth 2 when a "capabilityAction" is not in "allowedAction"
        ✓ should verify a capability chain of depth 2 and a specific expected capability action
        ✓ should fail to verify a capability chain of depth 2 when an expected "capabilityAction" is required but missing
        ✓ should fail to verify a capability chain of depth 2 when an expected "capabilityAction" does not match
        ✓ should verify a capability chain of depth 3
        ✓ should fail to verify a capability chain of depth 3 because of bad middle capability
        ✓ should fail to verify a capability chain of depth 3 because of bad last capability
        ✓ should verify a capability chain of depth 3 w/inspectCapabilityChain
        ✓ should fail to verify a capability chain of depth 3 w/inspectCapabilityChain that includes a revoked capability
        ✓ should verify invoking a capability chain of depth 3
        ✓ should verify a capability chain of depth 3 w/ multiple delegators
        ✓ should fail to verify a capability chain of depth 3 when delegation is not permitted
        ✓ should verify invoking a capability chain of depth 3 w/inspectCapabilityChain
        ✓ should fail invoking a capability chain of depth 3 w/inspectCapabilityChain and a revoked capability


  42 passing (656ms)

```